tests/framework/integration/cluster.go

```diff
@@ -37,6 +37,8 @@ import (
 	"go.uber.org/zap/zapcore"
 	"go.uber.org/zap/zaptest"
 
+	"go.etcd.io/raft/v3"
```

Comment:
Please manually execute `./scripts/fix.sh`

Reply:
The script does not generate new code changes. And the CI has passed. 

raft is splited into another repository.. So my IDE auto separate it to another group. 

<img width="695" alt="image" src="https://user-images.githubusercontent.com/54131596/226068193-797ecd94-69ad-4515-849e-fedeffbf0c89.png">

Reply:
thx for the confirmation.  The reason why I asked this is your dev branch is far behind the head, and do not include the PR https://github.com/etcd-io/etcd/pull/15485.

Reply:
aha, thanks for pointing it out.